### PR TITLE
make the how-to-promote guide a bit more modern

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: howto-promote.dd [hunk 1]
```diff
@@ -94,4 +91,3 @@ $(H3 Promoting the Project)
 Macros:
 	TITLE=Promoting D Projects (or Internet Marketing 101)
 	WIKI=PromotingDProjects
```
Comment (wilzbach) [inline howto-promote.dd]: for what is this macro used?

Comment (CyberShadow) [inline howto-promote.dd]: It's no longer used. Previously each page had a "page wiki" link, which lead to an associated page on the wiki, where users could add notes.

**Comment by wilzbach** [comment]
Rebased after the wiki changes. In my second pass I also took the courage to modify the list of old links and websites.

**Comment by CyberShadow** [comment]
There's some DDoc peeking out:

```
 $(D_S Promoting D Projects (or Internet Marketing 101), 
```

**Comment by wilzbach** [comment]
> There's some DDoc peeking out:

It was a missing parenthesis somewhere :D

**Comment by wilzbach** [comment]
> I didn't even know this page existed, but it's certainly an improvement. ;)

I also found it by accident, but the plan is to show a link to this when one creates a new announce thread.
See also https://github.com/CyberShadow/DFeed/issues/69

**Pull request merged**